src/video/sdl2_v.cpp

```diff
@@ -500,9 +505,6 @@ static const char *InitializeSDL()
 	 * UpdateWindowSurface() to update the window's texture instead of
 	 * its surface. */
 	SDL_SetHint(SDL_HINT_FRAMEBUFFER_ACCELERATION, "0");
-#ifndef __EMSCRIPTEN__
-	SDL_SetHint(SDL_HINT_MOUSE_RELATIVE_MODE_WARP, "1");
-#endif
```

Comment:
Why this removal?

Reply:
Because leaving it may give a false information that mouse relative mode wrap is defined solely by the target platform (anything not WebAssembly if I am correct?), whereas this patch overrides it at runtime based on the SDL version check.